app/views/admin/settings/index.html.slim

```diff
@@ -32,6 +32,10 @@ section.block
               = ldate(@last_updated_at, format: :long)
             - else
               = t("quickbooks.connection.never_synced")
+        = f.input "qb_data_import" do
+          .read-only
+            - if @fetch_task.present?
```

Comment:
if we get here, fetch_task should be present except in cases when there's a qb connection in the db that is not actually still live. I think we're moving towards avoiding those cases, and they are rare and usually in testing situations. There's no `else` logic because that case shouldn't happen, but still need to check to avoid an error that prevents you from resetting.